registry/servicediscovery/consul/go-client/pkg/user.go

```diff
@@ -15,25 +15,13 @@
  * limitations under the License.
  */
 
-package main
+package pkg
```

Comment:
maybe u can change the dir name. 'pkg' is not so good.

Reply:
all samples use pkg ,  so do i